src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGeneratorTest.java

```diff
@@ -38,14 +39,17 @@
 
     private static DetailAST rootAst;
 
+    private static String fileName;
+
     @Override
     protected String getPackageLocation() {
         return "com/puppycrawl/tools/checkstyle/xpath/xpathquerygenerator";
     }
 
     @Before
```

Comment:
can we use BeforeClass annotation?

Reply:
yes, but then we should make `init` method `static`, and if it is static then we can not call `getPath` method

Reply:
Ok, forgot that we make getPath method related to the object.

Reply:
It looks like if I use "Before" annotation instead of "BeforeClass", code seems to be cleaner, since I do not have to specify full path explicitly inside `init` method

Which one to use

Reply:
OK, lets stay on Before